flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/expressions/ScalarFunctionsTest.scala

```diff
@@ -2373,6 +2373,14 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
 
   @Test
   def testToTimestamp(): Unit = {
+    testAllApis(
```

Comment:
Would be also nice to have negative tests like with wrong input, with different mask